ledger/ledger-api-common/BUILD.bazel

```diff
@@ -29,6 +30,9 @@ da_scala_library(
     runtime_deps = [
         "@maven//:io_netty_netty_tcnative_boringssl_static",
     ],
+    scalacopts=[
+        "-P:silencer:globalFilters=DeduplicationTime.*deprecated",  # deprecated field that needs to be tested
```

Comment:
I'm not convinced that this is the right PR to also remove usage of deprecated stuff from code I don't know which is why I ignored the warning. Maybe someone can weigh in here and take care of it?